test/integration/features/addon-registry-route.feature

```diff
@@ -0,0 +1,45 @@
+@addon-registry-route @addon
+Feature: registry-route add-on
+registry-route exposes private docker registry,
```

Comment:
`registry-route` addon exposes the route for the OpenShift integrated registry so that user can able to push their container images directly to registry and deploy it.`

Reply:
Done